CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### master
```

Comment:
Thanks for tracking this here. I think I should add other stuff that I've changed too...